### DISCUSSION TITLE
Jellyfin 10.9.1 How to add lyrics to a song?

### DISCUSSION THREAD

**Discussion by 1P1E10** [jellyfin/jellyfin#11649] [General]
What's the instructions to add lyrics to a song? Do I have to use a separate lrc file or Jellyfin has built-in feature that get the lyrics for me?

**Comment by olip-03** [comment]
Probably have to wait for someone to make a plugin that does it for you, until then you might want to look at lrcget https://github.com/tranxuanthang/lrcget

**Comment by crobibero** [comment]
I have a plugin here: https://github.com/crobibero/jellyfin-plugin-lrclib

I just forgot to make sure Jellyfin would actually use it during a scan. (https://github.com/crobibero/jellyfin-plugin-lrclib/issues/3)

**Reply by nixietab** [reply]
do you think to solve the problem?

**Comment by Felitendo** [comment]
You could use my Plugin that automatically downloads the Lyrics to all songs for you: https://github.com/Felitendo/jellyfin-lyrics-plugin

**Reply by 1P1E10** [reply]
Thank you!

**Reply by Veto8162** [reply]
it is integrated into official jellyfin plugins [repo](https://github.com/jellyfin/jellyfin-plugin-lrclib)
